pom.xml

```diff
@@ -23,9 +23,7 @@
 
   <properties>
     <!-- jboss-parent override -->
-    <maven.compiler.target>1.8</maven.compiler.target>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <version.javadoc.plugin>3.4.0</version.javadoc.plugin>
+    <maven.compiler.release>8</maven.compiler.release>
```

Comment:
Somewhere in here we need to define the `nexus.repository.release` property to `arquillian`. There is a JBoss Nexus repository specifically for it, https://repository.jboss.org/nexus/repository/arquillian/.

We don't currently have a staging repository, but we could request one gets created or we find a shared staging repository with validation.

Reply:
@jamezp Are you sure? Because last time I looked it was empty - https://repository.jboss.org/nexus/#browse/browse:arquillian

Reply:
Yes, it's empty because we've not pushed anything to it :) 

If we don't use staging, which is okay, we need to make sure we're setup to push to Maven Central. IoW we need to make sure we don't have any empty name or description tags and the URL and SCM are correct. There might be some other things too.

There is a task that runs daily to deploy to Maven Central. IMO when we deploy we should run that task as a dry-run to ensure we'll pass validation. This is something I can do when we're ready.

Reply:
The solution with little downside is trivial - release a CR1. We have done that for other projects with no issues.

Reply:
Added the `arquillian` repository, however I am still not sure what are the benefits of it.